databases/week3/main.sql

```diff
@@ -0,0 +1,225 @@
+CREATE DATABASE mealSharing
+DEFAULT CHARACTER SET='utf8mb4';
+
+use mealSharing;
+
+CREATE Table meal(
+    id INT NOT NULL AUTO_INCREMENT  PRIMARY KEY,
+    title VARCHAR(50) NOT NULL,
+    description TEXT(1000) NOT NULL,
+    location VARCHAR(100) NOT NULL,
+    delivering_time DATETIME NOT NULL,
+    max_reservations INT(10) NOT NULL ,
+    price DECIMAL(10) NOT NULL,
+    created_date DATE
+);
+
+CREATE Table reservation(
+    id INT NOT NULL AUTO_INCREMENT  PRIMARY KEY,
+    number_of_guests int(10) NOT NULL,
+    created_date DATE ,
+     meal_id int(10),
+    FOREIGN KEY (meal_id) REFERENCES meal(id),
```

Comment:
could have used the CONSTRAINT statement to define the foreign keys, it allow for extra definitions regarding foreign keys, Its also a best practice.